frontend/src/components/App/App.tsx

```diff
@@ -69,7 +69,7 @@ const App = () => {
                         {/* Default experiment */}
                         <Route path="/" exact>
                             <Redirect
-                                to={URLS.block.replace(":slug", EXPERIMENT_SLUG)}
+                                to={URLS.experimentCollection.replace(":slug", EXPERIMENT_SLUG)}
```

Comment:
Isn't `EXPERIMENT_SLUG` related to what's now called a `Block`?

Reply:
I figured the default experiment slug as set in the environment variables will now actually be the slug of an experiment(collection) like `toontjehoger`, or what do you think?

Reply:
I'm merging this. I could do something with `EXPERIMENT_SLUG` in a later PR if needed :-)